src/NexusMods.DataModel.SchemaVersions/NexusMods.DataModel.SchemaVersions.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <RootNamespace>NexusMods.DataModel.Migrations</RootNamespace>
```

Comment:
What's up with this?

Reply:
I think it's something with a template that created this file, deleting it